src/fr/2017-03/08/06.md

```diff
@@ -1,7 +1,14 @@
 ---
-title:  Leçon quotidienne
+title:  Pourquoi retourner à l’esclavage? (Gal. 4:8–20).
```

Comment:
@clope031  You can not have `:` in the title without escaping it, because the meta portion is `yaml` format.
We need to either escape with single ``` ` ``` like ``` `Pourrquoi retourner ` ``` or in this case move the verse reference below